virttest/utils_misc.py

```diff
@@ -3550,34 +3560,47 @@ def get_pci_path(pci_id, session=None):
     return pci_path.strip()
 
 
-def bind_device_driver(pci_id, driver_type):
+def bind_device_driver(pci_id, driver_type, session=None):
     """
     Bind device driver.
 
+    :param pci_id: pci id of a device
+    :param session: ShellSession object of VM or remote host
     :param driver_type: Supported drivers: igb, lpfc, vfio-pci
     """
-    vd_list = get_pci_vendor_device(pci_id)
+    vd_list = get_pci_vendor_device(pci_id, session=session)
     if len(vd_list) == 0:
         LOG.error("Can't find device matched.")
         return False
     bind_file = "/sys/bus/pci/drivers/%s/new_id" % driver_type
     vendor = vd_list[0].split(":")[0]
     device = vd_list[0].split(":")[1]
     bind_cmd = "echo %s %s > %s" % (vendor, device, bind_file)
-    return process.run(bind_cmd, ignore_status=True, shell=True).exit_status == 0
+    status, output = cmd_status_output(bind_cmd, shell=True, session=session)
```

Comment:
I think it also should be the same as the above changed since `cmd_status_output` does not raise any error.

Reply:
So there is "if status != 0" in the following code is used to handle the exception. and lots other function like" def get_pid " has similar usage.

Reply:
> So there is "if status != 0" in the following code is used to handle the exception. and lots other function like" def get_pid " has similar usage.

Hi , @quanwenli 
My fault, I missed the argument `ignore_status=True`.